src/component/common/Table/TableActions/TableSearchField/TableSearchField.styles.ts

```diff
@@ -0,0 +1,39 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles(theme => ({
+    container: {
+        display: 'flex',
+        alignItems: 'center',
+        flexWrap: 'wrap',
+        gap: '1rem',
+    },
+    search: {
+        display: 'flex',
+        alignItems: 'center',
+        // @ts-expect-error
```

Comment:
This should be solved by augmenting the theme type instead of setting `// @ts-expect-error`

Reply:
Wasn't sure how to extend the theme to add this property, since all the current examples use `// @ts-expect-error`, so I made the change to `theme.palette.background.paper`, which maybe makes more sense, since we want this to be the color of the background paper, whatever that color is (and not something like `theme.palette.common.white` statically).